driver/normalizer/normalizer.go

```diff
@@ -156,7 +300,6 @@ var Normalizers = []Mapping{
 		Obj{
 			"Identifier": Obj{
 				uast.KeyType: String("IdentifierToken"),
-				// TODO(dennwc): assert that it's the same as in parent
```

Comment:
Well, yes, I also check, it *looks like* they are the same. But the "assertion" part in this TODO was to actually to make the transform to validate it :)

Not a release blocker, of course, but I'd like to keep those comments.

Reply:
Not sure how to do this (compare a node pos with a parent node pos), could you commit the change?

Reply:
This won't matter now: I converted the `IdentifierToken` nodes to `uast.Identifier` instead of the `IdentifierNames` (since it's the node actually holding the identifier and simplifies a lot of things).